benchmarks/Benchmarks.csproj

```diff
@@ -7,6 +7,7 @@
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)/../build/debug.snk</AssemblyOriginatorKeyFile>
     <DefineConstants>$(DefineConstants);SIGNED</DefineConstants>
+    <TieredCompilation>false</TieredCompilation>
```

Comment:
Is it needed? I think Benchmarkdotnet warms up iterations to avoid tiered compilation

Reply:
This ended up being vital to debugging/optimizing the code as a result of benchmarks, not necessarily running the benchmarks themselves. Here's the discussion I had with the runtime guys about it: https://github.com/dotnet/runtime/issues/33133

Thought it might help prevent future headaches to have in the project? Easy enough to remove.